sql/registrations_clean_up.sql

```diff
@@ -2,7 +2,7 @@ UPDATE hpd_registrations SET streetname = regexp_replace( streetname, ' AVE$|-AV
 
 
 -- unsure what to do about the conflict with streets?
```

Comment:
did you notice any issues with streets? at some point we should figure out a better solution to standardizing addresses instead of all these regexs